cli/ballerina-cli/src/main/resources/cli-help/ballerina-encrypt.help

```diff
@@ -6,8 +6,8 @@ SYNOPSIS
 
 
 DESCRIPTION
```

Comment:
I don't think this command is there anymore. @warunalakshitha

Reply:
Yes. We need to remove this. Seems missed once we removed the encrypt command.

Reply:
Removed the `ballerina-encrypt.help` file.